build.gradle

```diff
@@ -38,4 +40,65 @@ dependencies {
 
 test {
     useJUnitPlatform()
-}
\ No newline at end of file
+}
+
+
+task javadocJar(type: Jar) {
+    classifier = 'javadoc'
+    from javadoc
+}
+
+task sourcesJar(type: Jar) {
+    classifier = 'sources'
+    from sourceSets.main.allSource
+}
+
+artifacts {
+    archives javadocJar, sourcesJar
+}
+
+group = "com.zopa"
+archivesBaseName = "ktor-opentracing"
+version = "0.1.0"
```

Comment:
What happens in terms of versioning? Is this hard-coded?

Reply:
yes it's hard-coded